app/assets/stylesheets/components/_blue-box-campaign.scss

```diff
@@ -0,0 +1,43 @@
+$light-grey: #bfc1c3;
+$blue: #005ea5;
+$white: #ffffff;
+
+.blue-box-campaign {
```

Comment:
These component styles should be prefixed with `app-c-` as they are app level components.